tests/tensorflow/test_tensorflow2_autolog.py

```diff
@@ -146,6 +137,15 @@ def clear_fluent_autologging_import_hooks():
     mlflow.utils.import_hooks._post_import_hooks.pop("keras", None)
 
 
+@pytest.fixture(autouse=True)
+def clear_autologging_config():
+    """
+    Clears TensorFlow autologging config, simulating a fresh state where autologging has not
+    been previously enabled with any particular configuration
+    """
+    AUTOLOGGING_INTEGRATIONS.pop(mlflow.tensorflow.FLAVOR_NAME, None)
```

Comment:
+1 to explicitly clearing the dict key instead of AUTOLOGGING_INTEGRATIONS.clear()
also, clever state management approach.